librosa/core/audio.py

```diff
@@ -712,3 +712,66 @@ def clicks(times=None, frames=None, sr=22050, hop_length=512,
             click_signal[start:end] += click
 
     return click_signal
+
+
+def tone(frequency, sr=22050, length=None, duration=None, phi=90):
```

Comment:
(Redundant note here, just for maintaining GH status)

1. phi should be in radians, not degrees
2. default should be `None`
3. `phi=None` should map to `-np.pi/2` (not `+np.pi/2`)